tests/test_regex_identifier.py

```diff
@@ -448,6 +448,60 @@ def test_youtube_channel_id():
     _assert_match_first_item("YouTube Channel ID", res)
 
 
+def test_twitter():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["https://twitter.com/jack/status/20"], boundaryless=Filter({"Tags": ["Media"]}))
```

Comment:
Could you remove the boundaryless part in every test since it is not necessary?

Reply:
Sure thing, done